boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -174,11 +174,26 @@
 		#size-cells = <1>;
 
 		cpuapp_slot0_partition: partition@a6000 {
-			reg = <0xa6000 DT_SIZE_K(512)>;
+			reg = <0xa6000 DT_SIZE_K(488)>;
```

Comment:
If this needs to change, then please update `nrf54h20dk_nrf54h20_cpuapp.yaml` as well.

Reply:
If my calculations are correct, the `cpuapp_rw_partitions` is placed 552 KB further than this partition. So there is no need to shrink it. It could be even extended to 552 KB if needed.

BTW, is there a reason for this 64kB hole between the regions?